src/lib.rs

```diff
@@ -46,10 +46,11 @@
 //! [not sufficient on some platforms]: https://ts.data61.csiro.au/projects/TS/cachebleed/
 
 #![no_std]
-#![cfg_attr(all(test, feature = "nightly"), feature(test))]
+#![cfg_attr(all(test, feature = "nightly"), feature(test, bench_black_box))]
```

Comment:
`bench_black_box` was stabilised in 1.66. (But might be good to keep for people trying with an older toolchain)